pkg/services/apiserver/service.go

```diff
@@ -258,6 +262,36 @@ func (s *service) start(ctx context.Context) error {
 			return err
 		}
 
+	case grafanaapiserveroptions.StorageTypeUnifiedNext:
+		if !s.features.IsEnabledGlobally(featuremgmt.FlagUnifiedStorage) {
+			return fmt.Errorf("unified storage requires the unifiedStorage feature flag")
+		}
+
+		server, err := entitybridge.ProvideResourceServer(s.db, s.cfg, s.features, s.tracing)
+		if err != nil {
+			return err
+		}
+		serverConfig.Config.RESTOptionsGetter = apistore.NewRESTOptionsGetterForServer(server,
+			o.RecommendedOptions.Etcd.StorageConfig.Codec)
+
+	case grafanaapiserveroptions.StorageTypeUnifiedNextGrpc:
+		if !s.features.IsEnabledGlobally(featuremgmt.FlagUnifiedStorage) {
+			return fmt.Errorf("unified storage requires the unifiedStorage feature flag")
+		}
+		// Create a connection to the gRPC server
+		conn, err := grpc.NewClient(o.StorageOptions.Address, grpc.WithTransportCredentials(insecure.NewCredentials()))
+		if err != nil {
+			return err
+		}
+
+		// TODO: determine when to close the connection, we cannot defer it here
+		// defer conn.Close()
```

Comment:
I'm not 100% sure about the lifetime of this, but I guess we could use `stoppedCh` in a go routine to close it gracefully close when we exit.

Reply:
I am thinking the same. This is copy paste from the previous entity-server code .